core/input/input_event.cpp

```diff
@@ -837,6 +861,11 @@ void InputEventMouseButton::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "button_index"), "set_button_index", "get_button_index");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "pressed"), "set_pressed", "is_pressed");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "double_click"), "set_double_click", "is_double_click");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "input_event_state"), "set_state", "get_state");
```

Comment:
I think this property should be exposed in the base class (`InputEvent`) and not multiple times in multiple child classes. Since it is actually in the `InputEvent` class and the getter is already exposed.